### DISCUSSION TITLE
brew install efl [optional dep QT -> build error]

### DISCUSSION THREAD

**Discussion by 2246sam20** [Homebrew/discussions#1907] [Getting started]
Hello!
well someone from the EFL org community said that qt is not needed for EFL directly or indirectly.
Since m getting errors when brew tries to install qt.
Can I somehow disable qt so that brew doesn't install it??

Refer to the below thread  to know about the error 
`https://github.com/Homebrew/discussions/discussions/1888#discussion-3477174 `

**Comment by SMillerDev** [comment]
brew defines the requirement as follows: `efl -> poppler -> qt`. If you or the EFL developers believe that this is not required, a PR can be made against homebrew-core to rectify this.

**Reply by 2246sam20** [reply]
@SMillerDev  m not the one saying this,  EFL developers said this.
I discussed with them on their contact page.
Someone said that qt is not required directly or indirectly for EFL.

**Reply by carlocab** [reply]
You'll have to remove the `poppler` dependency then, or build `poppler` without depending on `qt`. You'll need to maintain your own versions of these formulae for this.

Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/
